### PR TITLE
mpi4py 2.0.0 version for python 2.7.12 and intel 2016b

### PR DESCRIPTION
**Pull request by jhein32**
Second attempt of a PR into the EB main develop branch

### CODE REVIEW

**Comment by jhein32** [comment]
@boegel Hmm,I tried to do this in github desktop on my laptop.  I issued a pr from my private development branch into hpcugent/develop and thought that was yours.  Perhaps I am still missing something or the github interface is a bit misleading.

I now issued the PR on the website and travis got active

File: easybuild/easyconfigs/m/mpi4py/mpi4py-2.0.0-intel-2016b-Python-2.7.12.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+source_urls = [BITBUCKET_DOWNLOADS]
+sources = [SOURCE_TAR_GZ]
+
+py_maj_min = '2.7'
+pyver = '%s.12' % py_maj_min
+versionsuffix = '-Python-%s' % pyver
+
+dependencies = [('Python', pyver)]
```
Comment (boegel) [inline easybuild/easyconfigs/m/mpi4py/mpi4py-2.0.0-intel-2016b-Python-2.7.12.eb]: @jhein32 please update this to use the `%(pyver)s` template:

```python
name = 'mpi4py'
version = '2.0.0'
versionsuffix = '-Python-%(pyver)s'
...
dependencies = [('Python', '2.7.12')]
...
    'dirs': 'lib/python%(pyshortver)s/site-packages/mpi4py'],
```

you can add a new commit to this branch and push the updated branch to update this PR

File: easybuild/easyconfigs/m/mpi4py/mpi4py-2.0.0-intel-2016b-Python-2.7.12.eb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+homepage = 'https://bitbucket.org/mpi4py/mpi4py'
+description = """MPI for Python (mpi4py) provides bindings of the Message Passing Interface (MPI) standard for
+ the Python programming language, allowing any Python program to exploit multiple processors."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+source_urls = [BITBUCKET_DOWNLOADS]
+sources = [SOURCE_TAR_GZ]
+
+dependencies = [('Python', '2,7.12')]
```
Comment (boegel) [inline easybuild/easyconfigs/m/mpi4py/mpi4py-2.0.0-intel-2016b-Python-2.7.12.eb]: @jhein32 typo here, should be `2.7.12` (Travis will tell you ;))

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2607.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/dbf881e9d2e96d42e7bc825e8cca2df9 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node812.raichu.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/afe18118ff1aab4ae7dcf30c38dbf223 for a full test report.

**Comment by boegel** [comment]
lgtm, will merge when Travis gives the green light

Thanks @jhein32!

**Comment by boegel** [comment]
Going in, thanks @jhein32!

**Pull request merged**